api.ts

```diff
@@ -26,7 +26,6 @@ export * from './api/events-api';
 export * from './api/files-api';
 export * from './api/health-api';
 export * from './api/onboarding-api';
-export * from './api/outbound-transfers-api';
```

Comment:
@BMayhew  @leftwichj It looks like removing outbound-transfers from the spec removed it from the SDK. I don't see any SDK requests using the SDK over the last 30 days in [sandbox](https://tilled.sumologic.com/ui/#/search/create?id=JSAWy5zvo3WGxn9rVh9rKZtPk647ik7PuHsQB95M) or [production](https://tilled.sumologic.com/ui/#/search/create?id=w4mt46VCS1WXb6Db7pqqdJTimzhiyTVHsTI7U90z), but I am going to hold off on merging this for now.

Reply:
sounds good!